huxley/www/js/components/Huxley.js

```diff
@@ -49,7 +50,11 @@ var Huxley = React.createClass({
   render: function() {
     var user = CurrentUserStore.getCurrentUser();
     if (User.isAnonymous(user)) {
-      return React.cloneElement(this.props.children, { user: user });
+      return (
+        <Shaker>
```

Comment:
Will this shake the whole screen or just the view that contains our forms (e.g. LoginView, RegistrationView)? And does it make a difference either way?